cheat_sheets/pydpf-core_cheat_sheet/pydpf-core_cheat_sheet.tex

```diff
@@ -0,0 +1,203 @@
+\documentclass[9pt,landscape]{article}
+\usepackage{{./../static/style}}
+\pdfinfo{
+  /Title (PyDPF-Core Cheat Sheet)
+  /Creator (TeX)
+  /Producer (pdfTeX 1.40.0)
+  /Author (Ansys)
+  /Subject (PyDPF)
+  /Keywords (PyAnsys, PyDPF, DPF, Cheat sheet, template)}
+
+\begin{document}
+\raggedright
+\footnotesize
+
+% Add the title of cheat sheet here
+% ----------------------------------------
+
+\begin{center}
+     \Huge{\textbf{PyDPF-Core Cheat sheet}} \\
+\end{center}
+\AddToShipoutPicture*
+  {\put(670,577.5){\includegraphics[height = 1.2cm]{ansys.png}}}
+\AddToShipoutPictureBG*{\includegraphics[width=\paperwidth]{bground.png}}
+\vspace{-0.15cm}
+\noindent\makebox[\linewidth]{\rule{\paperwidth}{2pt}}
+
+\begin{multicols}{3}
+\setlength{\premulticols}{1pt}
+\setlength{\postmulticols}{1pt}
+\setlength{\multicolsep}{1pt}
+\setlength{\columnsep}{2pt}
+
+% session starts here. 
+% First colomn
+% --------------------------------------------------------------------------------
+
+\section{\includegraphics[height=\fontcharht\font`\S]{slash.png} Importing a Model}
+The DPF Model gives an overview of the result files content.
```

Comment:
```suggestion
The DPF Model gives an overview of the result files content
```
We can go with consistency, with period or without. I personally like without period in this case. Please adapt all the inline docstring accordingly.

Reply:
Other cheat-sheets also have period, so using period consistently in here as well.